### PR TITLE
Swift doc cleanup (SOC-8649)

### PR DESCRIPTION
**Pull request by dmpop**
Add entries for adding and removing disk drives.

### CODE REVIEW

File: xml/operations-objectstorage-input_model_change_existing_rings.xml [hunk 1]
```diff
@@ -417,6 +436,20 @@ the required actions. More detailed procedures are provided elsewhere.-->
...
      </row>
+     <row>
+      <entry>
+       Removing Disk Drive(s)
+      </entry>
+      <entry>
+       <para>
+	When removing a disk drive from the input model, keep in mind that this
+	drops the disk out of the ring without allowing Swift to move the data
+	off it first. While it should fine in a properly replicated healthy
```
[the remark this answers is not in the document]
Comment (dmpop) [reply xml/operations-objectstorage-input_model_change_existing_rings.xml]: Fixed. Thanks for spotting this!

File: xml/operations-objectstorage-input_model_change_existing_rings.xml [hunk 2]
```diff
@@ -417,6 +436,20 @@ the required actions. More detailed procedures are provided elsewhere.-->
...
+     <row>
+      <entry>
+       Removing Disk Drive(s)
+      </entry>
+      <entry>
+       <para>
+	When removing a disk drive from the input model, keep in mind that this
+	drops the disk out of the ring without allowing Swift to move the data
+	off it first. While it should be fine in a properly replicated healthy
+	cluster, this approach is not recommended. A better solution is to step
```
Comment (asettle) [inline xml/operations-objectstorage-input_model_change_existing_rings.xml]:
```suggestion
cluster, we do not recommend this approach. A better solution is to step
```

**Comment by matthewoliver** [comment]
Yeah sorry, I keep meaning to get to it and then my day ends. First thing
tomorrow I promise.. after the swift meeting and standup meeting!

**Comment by asettle** [comment]
It's broken because of this PR https://github.com/SUSE-Cloud/doc-cloud/pull/1153/commits/890118dfaaed0861b82895f839c574d20029898a didn't follow the conventions. I'll rename and push.

**Comment by asettle** [comment]
Conflicts galoreeeeeeee - that was me, I will fix that :p

**Pull request closed without merging**